src/views/mainLayout.js

```diff
@@ -136,26 +136,26 @@ const userMenu = data => `
         <b class='user-menu-email'>${data.fxaProfile?.email}</b>
         <div class='user-menu-subtitle'>
           ${getMessage('menu-item-fxa')}
-          <img src='/images/icon-open-in.svg' />
+          <img src='/images/icon-open-in.svg' alt='Open up icon' />
```

Comment:
Nit: I think all of these `alt="..."` attributes would need to be localized.
Also, should it be "Open **_in_** icon" vs "Open _up_ icon" (based on filename)?